### ISSUE TITLE
test issue: stems_and_words

### ISSUE THREAD

**Issue by JohnAGoldsmith** [axanthos/pycrab#5]
for "serialize_stems_and_words" 

We discussed marking affixes with a ":"(5:prefix  or suffix:5) and therefore stems  can be identified as having no ":"
Line799 can then be changed, so it finds all stems in each parse of a word (and a word can have two stems if it is a compound):

**Comment by axanthos** [comment]
Yes, and we will have to review every use of stems in the code in case they are affected by the change. The first thing to do is probably to implement "labels" in parses.
